test/Query.test.ts

```diff
@@ -77,15 +77,21 @@ describe('queryJSON', () => {
 });
 
 it('map', async () => {
```

Comment:
I think this test now has 2 different responsibilities and thus I would split it in something like:
- `it("when a mapped query resolves the expected value is returned", ...)`
- `it("when a mapped query is invalidated all its components are recomputed", ...)`

Reply:
Makes sense, I'll do